src/components/Advertising/Advertising.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react';
+import s from './Advertising.module.css';
+
+import { images } from './images/index';
+
+const Advertising = () => {
+  const [currentImageIndex, setCurrentImage] = useState()
+
+  const randomNumber = (Math.floor(Math.random() * images.length));
+
+  const changeImageIndex = () => {
+    setCurrentImage(randomNumber)
+  }
+
+  useEffect(
+    () => changeImageIndex(),
+    []
+  );
+
+
+  return (
+    <div className={s.hero}>
+      <img
+        className={s.image}
+        src={images[currentImageIndex]}
```

Comment:
```suggestion
        src={images[randomNumber]}
```
Why not just pass a random number here without useEffect and useState hooks?

Reply:
This is used if in the future Ads will be loaded from the backend